.github/workflows/preview-theme.yml

```diff
@@ -7,6 +7,8 @@ on:
       - master
       - theme-preview-script
```

Comment:
@anuraghazra Why do you include the `theme-preview-script` and `themes/index.js` branches in the `pull_request_target`? Where do we use these?